tracks/rust/exercises/bob/mentoring.md

```diff
@@ -31,30 +31,30 @@ const CALM: &str = "Calm down, I know what I'm doing!";
 const FINE: &str = "Fine. Be that way!";
 const WHATEVER: &str = "Whatever.";
 
-#[allow(clippy::match_bool)]
 pub fn reply(message: &str) -> &str {
```

Comment:
While this has not been merged yet: a student added a comment that he changed the return type to `&'static str` since there is no sense in tying the output lifetime to the input. What are your thoughts? I don't really know, but I think it is both true and unnecessary. It is true that the output isn't tied to the input, but it is less clutter and it doesn't matter that there is no `'static` currently.

Reply:
And this comment makes me wonder, why a `&str` is returned at all, and not a `String` for the result.

Reply:
@NobbZ because the string doesn't need to be allocated on the heap; returning a `&str` is cheaper.

@L0uisc I agree that `&'static str` would make more sense. I'll ask it on slack, so we can discuss it there.